code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -18,6 +18,9 @@
 
 #define PUNCH_STAMINA_MULTIPLIER 2.6
 
+// Shove damage
+#define SHOVE_STAMINA_SOLID 10
```

Comment:
Considering how fast stamina is regenerated, I feel like this would make more sense, otherwise it'd take absolutely ages to get anything done from a shove.
```suggestion
#define SHOVE_STAMINA_SOLID 25
```